docs/journal/01-cold-storage.md

```diff
@@ -1,74 +1,90 @@
+## Common requirements
 
-![plan for this first milestone](media/01-cold-storage/logical-architecture-for-milestone01.png)
+Here are some common requirements for ingestion and cold storage that any IoT solution will at least need to consider:
+
+- Achieving scale. That means both hitting the expected target data rate, but also designing the system to be scalable in the future.
+- Authentication. Only authenticated devices can send data. (Provisioning devices is a separate scenario, although closely related.)
+- Protecting the data stream.
+- Storing the raw event data in long-term storage. Cold storage must hold a large volume of historical data indefinitely, at a reasonable cost. 
+
+  
+## Fabrikam requirements
+
+:memo: For more about our Fabrikam scenario, see [Introducing the Journey][00-intro]. 
 
+Here are some specific requirements for our scenario:
 
-## Devices
+- As an initial target, the system must handle up to 100,000 devices, each device sending 1 reading every minute. That's approximately 1,667 events per second. This is a minimum load &mdash; the system should be scalable beyond this target. 
+- Events might be time critical, so the Event Processing component must handle this volume of data without introducing any backlog into the system.
+- The event data stream must be protected. This means not only securing the content of the events, but also masking the *absence* of events. For example, if the motion detectors in a house are not streaming any events, an attacker can conclude that nobody is home.
 
-Initially, Fabrikam don't have access to the buildings, so they have elected to build a simple simulator that emits temperature events representing the readings for each device. These events are generated at a realistic rate, and the simulator can be configured to imitate a variable number of devices. The simulator has already been developed using the .NET Framework. A console application provides a simple user interface, but it is also possible to host the simulator as a web job to provide an environment for long-lived, unattended running.
 
-> ![Markus](media/PersonaMarkus.png) 
-"We invested a significant amount of time and effort in creating the simulator. It is important that we are able to simulate a realistic number of devices sending events at the expected rate."
+## Exploring solutions for ingestion and processing
 
-## Event Processing
+**Ingestion** and **event processing** are somewhat related. How the system ingests the events may limit the options for processing. We investigated the following options.   
 
-Per the customer requirements, the system needs to capture event information emanating from 100,000 devices, each device sending 1 reading every minute. This equates to approximately 1,667 events per second. The Event Processing component must be capable of capturing this volume of information without introducing any backlog into the system; events raised by devices could be time critical.
+- HTTP service
```

Comment:
In the intro here for ingestion, I would like to state that we chose (and encourage) the use of Event Hubs and that we are discussing the HTTP service and Service Bus solutions (not because you are likely to use them) but in order to clarify our reasoning. This different from other places where the best alternative (Stream Analytics vs Storm vs whatev) are more contextual.